src/main.ts

```diff
@@ -92,7 +92,8 @@ async function fetchContent(
   const response = await client.repos.getContents({
```

Comment:
Do we maybe want to retry on the default branch if this 404's? The scenario I'm concerned about is:
1) User X branches repo
2) This action gets merged into master
3) User X PRs, but is out of sync with master

No label would get applied at this point if they hadn't synced with master yet, even if they sync. I know this is kinda an edge case, but its most likely to crop up the first couple of times the action is run and isn't a great first experience.

Reply:
I'm not sure I'd agree with retrying on the default branch. My feeling is that workflows/actions should only run against the code in the ref/sha they are committed to. I think I'd be surprised if my branch used actions/configuration from a different branch.

I think the scenario you described is fairly unlikely and has an obvious remedy (rebase/merge master).

What do you think of shipping this as-is, and making the change if we get feedback from the community?